src/Bookings.js

```diff
@@ -1,19 +1,36 @@
-import React from "react";
+// ** Instructions:** In the`<Bookings />` component, declare a new state`bookings`
+//with the corresponding setter function `setBookings` to hold the`FakeBookings` data.
+//Instead of passing`FakeBookings` directly to the`<SearchResults />` component,
+//pass the new `bookings` state variable.
+
+// ** Hint:** The new `bookings` state should be initialised with the`FakeBookings`
+//variable.
+
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
+import SearchResults from "./SearchResults.jsx";
+
 // import SearchResults from "./SearchResults.js";
 // import FakeBookings from "./data/fakeBookings.json";
 
 const Bookings = () => {
+  const [BookingsData, setBokings] = useState([]);
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me`)
+      .then(res => res.json())
+      .then(data => setBokings(data));
+  }, []);
+  console.log(BookingsData);
```

Comment:
Well done with the useEffect hook! The console.log() is meant to be in scope of the useEffect hook though, where could we move it to?